### PR TITLE
as a user, i want to show number abandonments on gitcoin profile, so i can know who's a flake

### PR DESCRIPTION
**Pull request by owocki**
##### Description

as a user, i want to show number abandonments on gitcoin profile, so i can know who's a flake
[screenshot](http://bits.owocki.com/1t2m2N2G0w2K/Screen%20Shot%202018-05-29%20at%203.18.46%20PM.png)

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

profile

##### Testing

tested.. see screenshot
##### Refers/Fixes

Self

### CODE REVIEW

**Comment by owocki** [comment]
@PixelantDesign this is perhaps and aspect of the reputation system you'd wanted to buidl

**Comment by mbeacom** [comment]
Just tested this on staging with @SaptakS - It seems to be failing to allow me to boot someone that's started working:

```{errors: ["Party haven't expressed interest on this bounty."], success: false}
errors: ["Party haven't expressed interest on this bounty."]
success: false```

**Comment by owocki** [comment]
@mbeacom i dont disagree that thats a suboptimal UX, but i dont think that has anything to do with the changeset on this PR (which doesnt affect anything regarding the removal of bounty hunters from tasks; only functionally changes whats disaplayed on the profile)

**Comment by mbeacom** [comment]
@owocki Hhmm? I was only saying there is an error when attempting to remove the interest from a bounty in this PR as specified above.  Other than that, this implementation is fine for what we're attempting to do.

**Comment by owocki** [comment]
should we fix that issue in this PR or create a new issue?

File: app/dashboard/models.py [hunk 1]
```diff
@@ -1071,12 +1071,19 @@ def tips(self):
         tipped_for = Tip.objects.filter(username__iexact=self.handle).order_by('-id')
         return on_repo | tipped_for
 
-    def has_been_removed_by_staff(self):
+    def no_times_been_removed_by_staff(self):
         user_actions = UserAction.objects.filter(
             profile=self,
             action='bounty_removed_by_staff',
```
Comment (owocki) [inline app/dashboard/models.py]: well have to add the slash `action` to this too

**Pull request merged**